public/js/test/mochitest/examples/code_script-switching-02.js

```diff
@@ -0,0 +1,13 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+function secondCall() {
+  // This comment is useful: ☺
+  debugger;
+  function foo() {}
+  if (x) {
+    foo();
+  }
+}
+
+var x = true;
```

Comment:
I don't mind having several test cases available that do a few different things. I think this is a good one to have too.